laelf.adoc

```diff
@@ -696,14 +700,21 @@ with check 28-bit signed overflow and 4-bit aligned
 |`+(*(uleb128 *) PC) += S + A+`
 
 |108
-|R_LARCH_ADD_ULEB128
+|R_LARCH_SUB_ULEB128
 |ULEB128 in-place subtraction
 |`+(*(uleb128 *) PC) -= S + A+`
 
 |109
 |R_LARCH_64_PCREL
 |64-bit PC relative
 |`+(*(uint64_t *) PC) = (S+A-PC) [63 ... 0]+`
+
+|110
+|R_LARCH_CALL36
+|Used for medium code model function call instruction sequence pcaddu18i and jirl, these two instructions must adjacent.
```

Comment:
```suggestion
|Used for medium code model function call sequence `pcaddu18i + jirl`. The two instructions must be adjacent.
```

You could make the document prettier by making the other instruction mnemonics and other appropriate words `inline code blocks` too, while you're at it.

Reply:
Thanks, the next version will consider this.